core/templates/dev/head/pages/state_editor/StateContentEditorDirective.js

```diff
@@ -53,6 +53,21 @@ oppia.directive('stateContentEditor', [
 
           $scope.contentEditorIsOpen = false;
           $scope.isEditable = EditabilityService.isEditable;
+          $scope.isCardHeightLimitReached = false;
+          $scope.showCardHeightLimitWarning = true;
+
+          $scope.$watch(function() {
```

Comment:
Can we remove this watch once someone `hideCardHeightLimitWarning`, as we aren't going to show this message after that, right? (Sorry I missed this in the first review.)

Reply:
No issues, Done!